<?xml version="1.0" ?>
<testsuite errors="1" failures="0" name="test_demo.TestDemo1-20190118150031" tests="2" time="0.069">
	<testcase classname="test_demo.TestDemo1" name="test_002" time="0.000"/>
	<testcase classname="test_demo.TestDemo1" name="test_001" time="0.069">
		<error message="Message: element not interactable
  (Session info: chrome=71.0.3578.98)
  (Driver info: chromedriver=72.0.3626.7 (efcef9a3ecda02b2132af215116a03852d08b9cb),platform=Windows NT 10.0.16299 x86_64)
" type="ElementNotVisibleException">
<![CDATA[Traceback (most recent call last):
  File "cases/ui\test_demo.py", line 15, in test_001
    driver.find_element_by_xpath("//a[text()='登录']").click()
  File "C:\Users\qxv0514\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\webelement.py", line 80, in click
    self._execute(Command.CLICK_ELEMENT)
  File "C:\Users\qxv0514\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\webelement.py", line 633, in _execute
    return self._parent.execute(command, params)
  File "C:\Users\qxv0514\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Users\qxv0514\AppData\Local\Programs\Python\Python37\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.ElementNotVisibleException: Message: element not interactable
  (Session info: chrome=71.0.3578.98)
  (Driver info: chromedriver=72.0.3626.7 (efcef9a3ecda02b2132af215116a03852d08b9cb),platform=Windows NT 10.0.16299 x86_64)

]]>		</error>
	</testcase>
	<system-out>
<![CDATA[]]>	</system-out>
	<system-err>
<![CDATA[]]>	</system-err>
</testsuite>
